integration-tests/vault-app/src/main/java/io/quarkus/it/vault/VaultTestService.java

```diff
@@ -39,7 +39,7 @@ public String test() {
             return "someSecret=" + someSecret + "; expected: " + expectedPassword;
         }
         String password = ConfigProviderResolver.instance().getConfig().getValue("password", String.class);
-        if (!expectedPassword.equals(someSecret)) {
+        if (!password.equals(someSecret)) {
```

Comment:
This one makes sense but @vsevel could you have a look?

Reply:
the second case should be between `password` and `expectedPassword` (not between `password` and `someSecret`)
`password` and `someSecret` are calculated, `expectedPassword` is the reference (could be a constant)
so something like this:
```
        String password = ConfigProviderResolver.instance().getConfig().getValue("password", String.class);
        if (!expectedPassword.equals(password)) {
            return "password=" + password + "; expected: " + expectedPassword;
        }
```
thanks for catching that one.

Reply:
Thanks @vsevel! I changed the code according to your suggestion.